samples/subsys/usb/dfu/README.rst

```diff
@@ -0,0 +1,83 @@
+.. _usb_dfu:
+
+USB DFU Sample Application
+##########################
+
+Overview
+********
+
+This sample app demonstrates use of a USB DFU Class driver provided
+by the Zephyr project.
+
+Requirements
+************
+
+This project requires an USB device driver. Currently, the USB DFU
+class provided by the Zephyr project depends on DFU image manager and
+partition layout. Refer to :ref:`flash_partitions` for details about
+partition layout. You SoC have to run MCUboot as stage 1 bootloader.
+The sample will be build as application for the MCUboot bootloader.
+
+Building and Testing
+********************
+
+Building and signing the application
+====================================
+
+Sample can be built in the usual way (see :ref:`build_an_application`
+for more details) and flashed with regular flash tools, but will need
+to be loaded at the offset of SLOT-0.
+
+To boot the sample, the samle images must be signed.
+There is a `scripts\imgtool.py` script in the MCUboot source tree that
```

Comment:
Change to:

    Use the ``scripts/imagetool.py`` script from the `MCUboot GitHub repo`_  
    to sign the image.  (See the `Using MCUboot with Zephyr`_ documentation for
    details.)

.. _MCUboot GitHub repo: https://github.com/runtimeco/mcuboot
.. _Using MCUboot with Zephyr: https://mcuboot.com/mcuboot/readme-zephyr.html